### PR TITLE
updated l10n

### PR DESCRIPTION
**Pull request by Mirodin**

### CODE REVIEW

File: locale/de.po [hunk 1]
```diff
@@ -349,266 +381,134 @@ msgstr "Dokumente"
...
-msgstr "<b>Vorschau</b>"
+#: src/paperwork/frontend/mainwindow/mainwindow.glade.h:5
+msgid "Export"
+msgstr "Exportieren"
 
-#: src/paperwork/frontend/import/importfileselector.glade.h:1
-msgid "Select a file or a directory to import"
-msgstr "Eine Datei oder einen Ordner für den Import auswählen"
+#: src/paperwork/frontend/mainwindow/mainwindow.glade.h:6
+#, fuzzy
```
Comment (jflesch) [inline locale/de.po]: Please remove the tag "fuzzy" once you have checked the translation.

File: locale/de.po [hunk 2]
```diff
@@ -349,266 +381,134 @@ msgstr "Dokumente"
...
+msgid "Page"
+msgstr "Seite"
 
-#: src/paperwork/frontend/import/importaction.glade.h:1
-#: src/paperwork/frontend/mainwindow/mainwindow.glade.h:42
-msgid "Import file(s)"
-msgstr "Datei(en) importieren"
+#: src/paperwork/frontend/mainwindow/mainwindow.glade.h:7
+#, fuzzy
```
Comment (jflesch) [inline locale/de.po]: Same here (and all the followings)

**Comment by jflesch** [comment]
Almost perfect :)
If you can just remove those tags, I will merge it asap.

**Comment by Mirodin** [comment]
Sorry, did not know these were meant to be removed by me, I'll send another one.

**Pull request closed without merging**